sentry-ruby/lib/sentry/transport.rb

```diff
@@ -33,6 +49,23 @@ def send_event(event)
       event
     end
 
+    def is_rate_limited?(event_type)
+      # check category-specific limit
+      delay =
+        case event_type
+        when "event"
+          # confusing mapping, but it's decided by Sentry
```

Comment:
To shed some light on the decision behind this naming convention: Sentry understands several event types, among others "error" events (for error tracking) and "transaction" events (for performance).